t/cli/test_main.sh

```diff
@@ -152,7 +152,7 @@ if [ $count_http_specific_ip -ne 2 ]; then
     exit 1
 fi
 
-count_http_specific_ip_and_enable_http2=`grep -c "listen 127.0.0..:908. default_server http2" conf/nginx.conf || true`
+count_http_specific_ip_and_enable_http2=`grep -c "http2 on" conf/nginx.conf || true`
 if [ $count_http_specific_ip_and_enable_http2 -ne 1 ]; then
```

Comment:
Does this need to be split to 2 test cases? one for count_http_specific_ip and one for enable_http2

Reply:
`count_http_specific_ip` already exists. ref: https://github.com/apache/apisix/pull/10887/files/4eadeca705566f6db5772266c635b069444e7cd5#diff-cb891f9ef4943580aa92827c7d725f3c717923cb85344cc6d8195d70f7ef01b0L149